### PR TITLE
kb(grid):select/deselect on row click

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: knowledge-base/grid-select-or-deselect-item-on-row-click.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+title: Select or Deselect Items on Row Click in Grid
```
Comment (dimodi) [inline knowledge-base/grid-select-or-deselect-item-on-row-click.md:2]:
```suggestion
title: Select or Deselect Grid Items on Row Click
```

File: knowledge-base/grid-select-or-deselect-item-on-row-click.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+description: How to Select or Deselect Items on Row Click in Grid?
```
Comment (dimodi) [inline knowledge-base/grid-select-or-deselect-item-on-row-click.md:3]:
```suggestion
description: How to Select or Unselect Grid Rows on Row Click
```

File: knowledge-base/grid-select-or-deselect-item-on-row-click.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+type: how-to
+page_title: Select or Deselect Items on Row Click in Grid
```
Comment (dimodi) [inline knowledge-base/grid-select-or-deselect-item-on-row-click.md:5]:
```suggestion
page_title: Select or Deselect Grid Items on Row Click
```

File: knowledge-base/grid-select-or-deselect-item-on-row-click.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+## Description
+
+How can I select and deselect multiple rows by clicking on a row in a Grid with enabled selection?
+
+## Solution
+
+When selection is enabled in the Grid component, the built-in option for deselecting a row or selecting multiple rows is `Ctrl` + click. To deselect a row or select multiple rows by row clicking and without holding the `Ctrl` key, try the following:
+
+* Disable the built-in selection
```
Comment (svdimitr) [inline knowledge-base/grid-select-or-deselect-item-on-row-click.md:32]: Maybe some additional information can be applied here

File: knowledge-base/grid-select-or-deselect-item-on-row-click.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+</table>
+
+
+## Description
+
+How can I select and deselect multiple rows by clicking on a row in a Grid with enabled selection?
+
+## Solution
+
+When selection is enabled in the Grid component, the built-in option for deselecting a row or selecting multiple rows is `Ctrl` + click. To deselect a row or select multiple rows by row clicking and without holding the `Ctrl` key, try the following:
```
Comment (dimodi) [inline knowledge-base/grid-select-or-deselect-item-on-row-click.md:30]:
```suggestion
When Grid selection is enabled, the built-in option for deselecting a row or selecting multiple rows is `Ctrl` + Click (`Cmd` + Click on Macs). To deselect a row or select multiple rows by row clicking and without holding the `Ctrl` key, try the following:
```

**Review by dimodi — APPROVED** [review]

File: knowledge-base/grid-select-or-deselect-item-on-row-click.md (hunk 5 above)
Comment (svdimitr) [inline knowledge-base/grid-select-or-deselect-item-on-row-click.md:30]: Remove `try`, we are giving solutions, and we should be aware if they are working

Comment (ntacheva) [reply knowledge-base/grid-select-or-deselect-item-on-row-click.md]: Will do

File: knowledge-base/grid-select-or-deselect-item-on-row-click.md (hunk 4 above)
Comment (ntacheva) [reply knowledge-base/grid-select-or-deselect-item-on-row-click.md]: Done

**Pull request merged**